pkg/pgctl/pgrestore.go

```diff
@@ -76,3 +107,25 @@ func (x *Restore) SetOptions(o []string) {
 func (x *Restore) GetOptions() []string {
 	return x.Options
 }
+
+// DropSchemas drops all schemas except the system ones.
+func (x *Restore) DropSchemas() error {
+	dropSchemaSQL := `
+        DO $$ DECLARE
+            r RECORD;
+        BEGIN
+            FOR r IN (SELECT nspname FROM pg_namespace WHERE nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast') AND nspname !~ '^pg_temp_') LOOP
+                EXECUTE 'DROP SCHEMA IF EXISTS ' || quote_ident(r.nspname) || ' CASCADE';
+            END LOOP;
+        END $$;
+    `
+
+	cmd := exec.Command(PSQL, x.DsnUri, "-c", dropSchemaSQL)
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("error dropping schemas: %w", err)
+	}
```

Comment:
Let's handle these with loggers rather than sending directly to stdout/stderr. You could copy what's done in `(x *Dump) Exec` or send back (string, error)
```suggestion
        var result Result
	cmd := exec.Command(PSQL, x.DsnUri, "-c", dropSchemaSQL)
	stderrIn, _ := cmd.StderrPipe()
        go func() {
            result.Output = streamExecOutput(stderrIn, opts)
        }()
        cmd.Start()
        err := cmd.Wait()
        ...
```